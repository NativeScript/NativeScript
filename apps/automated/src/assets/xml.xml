<?xml version="1.0" encoding="UTF-8" ?>
<DocumentElement param="value">
  <First.Element some.attr="some.value">
    &#xb6; Some Text &#x00AE;
  </First.Element>
  <?some_pi some_attr="some_value"?>
  <SecondElement param2="something">
    Pre-Text <Inline>Inlined text</Inline> Post-text.
  </SecondElement>
  <entities>Xml tags begin with &quot;&lt;&quot; and end with &quot;&gt;&quot; Ampersand is &amp; and apostrophe is &apos;</entities>
  <script>
    <![CDATA[
function sum(a,b)
{
  return a+b;
}
]]>
  </script>
  <!--
  Hello,
  I am a multi-line XML comment.
-->
</DocumentElement>