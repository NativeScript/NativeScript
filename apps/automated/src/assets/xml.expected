{"eventType":"StartElement","position":{"line":2,"column":1},"elementName":"DocumentElement","attributes":{"param":"value"}}{"eventType":"StartElement","position":{"line":3,"column":3},"elementName":"First.Element","attributes":{"some.attr":"some.value"}}{"eventType":"Text","position":{"line":3,"column":41},"data":"\n    ¶ Some Text ®\n  "}{"eventType":"EndElement","position":{"line":5,"column":3},"elementName":"First.Element"}{"eventType":"StartElement","position":{"line":7,"column":3},"elementName":"SecondElement","attributes":{"param2":"something"}}{"eventType":"Text","position":{"line":7,"column":37},"data":"\n    Pre-Text "}{"eventType":"StartElement","position":{"line":8,"column":14},"elementName":"Inline"}{"eventType":"Text","position":{"line":8,"column":22},"data":"Inlined text"}{"eventType":"EndElement","position":{"line":8,"column":34},"elementName":"Inline"}{"eventType":"Text","position":{"line":8,"column":43},"data":" Post-text.\n  "}{"eventType":"EndElement","position":{"line":9,"column":3},"elementName":"SecondElement"}{"eventType":"StartElement","position":{"line":10,"column":3},"elementName":"entities"}{"eventType":"Text","position":{"line":10,"column":13},"data":"Xml tags begin with \"<\" and end with \">\" Ampersand is & and apostrophe is '"}{"eventType":"EndElement","position":{"line":10,"column":123},"elementName":"entities"}{"eventType":"StartElement","position":{"line":11,"column":3},"elementName":"script"}{"eventType":"CDATA","position":{"line":12,"column":5},"data":"\nfunction sum(a,b)\n{\n  return a+b;\n}\n"}{"eventType":"EndElement","position":{"line":18,"column":3},"elementName":"script"}{"eventType":"Comment","position":{"line":19,"column":3},"data":"\n  Hello,\n  I am a multi-line XML comment.\n"}{"eventType":"EndElement","position":{"line":23,"column":1},"elementName":"DocumentElement"}